templates/base.html

```diff
@@ -29,6 +29,7 @@
         {% endblock %}
     {% endblock %}
     {% block head-scripts %}{% endblock head-scripts %}
+    <script defer data-domain="next.okfn.org" src="https://plausible.io/js/plausible.js"></script>
```

Comment:
We'll need to change `data-domain` to `okfn.org` when going live

Reply:
I made [these changes](https://github.com/okfn/website/commit/f266a1170625e9db444abb2498f1b3cdcf33f615) to the PR @amercader
I think this way will be easy to add production with this